core/src/main/java/org/apache/spark/api/java/JavaFutureAction.java

```diff
@@ -17,7 +17,6 @@
 
 package org.apache.spark.api.java;
 
-
```

Comment:
This type of change shouldn't be made. Removing dead imports we usually don't bother with but maybe one sweep of all the code is OK.